crates/pop-contracts/src/new.rs

```diff
@@ -68,6 +70,9 @@ fn create_template_contract(
 	let template_repository = template.repository_url()?;
 	// Clone the repository into the temporary directory.
 	let temp_dir = ::tempfile::TempDir::new_in(std::env::temp_dir())?;
+	#[cfg(feature = "v5")]
+	Git::clone(&Url::parse(template_repository)?, temp_dir.path(), None)?;
```

Comment:
For ink! v5 fetch it from main: https://github.com/use-ink/ink-examples
For ink! v6 from: https://github.com/use-ink/ink-examples/tree/v6.x